src/js/msp/MSPConnector.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+var MSPConnectorImpl = function () {
+    this.baud = undefined;
+    this.port = undefined;
+    this.onConnectCallback = undefined;
+    this.onTimeoutCallback = undefined;
+    this.onDisconnectCallback = undefined;
+};
+
+MSPConnectorImpl.prototype.connect = function (port, baud, onConnectCallback, onTimeoutCallback, onFailureCallback) {
+
+    var self = this;
+    self.port = port;
+    self.baud = baud;
+    self.onConnectCallback = onConnectCallback;
+    self.onTimeoutCallback = onTimeoutCallback;
+    self.onFailureCallback = onFailureCallback;
+
+    serial.connect(self.port, {bitrate: self.baud}, function (openInfo) {
+        if (openInfo) {
+            var disconnectAndCleanup = function() {
+                serial.disconnect(function(result) {
+                    console.log('Disconnected');
+                    
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://app.codacy.com/gh/emuflight/EmuConfigurator/pullRequest?prid=5626377)